contracts/core/Dispatcher.sol

```diff
@@ -747,7 +739,7 @@ contract Dispatcher is OwnableUpgradeable, UUPSUpgradeable, ReentrancyGuard, IDi
         internal
         returns (bool success, bytes memory message)
     {
-        if (!Address.isContract(receiver)) {
+        if (receiver == address(0) || !Address.isContract(receiver)) {
```

Comment:
optimization?

Reply:
more so a completeness check; to avoid calling address(0) - maybe this isn't needed actually